layouts/default.vue

```diff
@@ -388,20 +392,20 @@
       <div class="buttons">
         <nuxt-link class="btn btn-outline btn-primary" to="/app">
           <DownloadIcon aria-hidden="true" />
-          {{ formatMessage(messages.getModrinthAppButton) }}
+          {{ formatMessage(messages.getModrinthApp) }}
         </nuxt-link>
         <button class="iconified-button raised-button" @click="changeTheme">
           <MoonIcon v-if="$colorMode.value === 'light'" aria-hidden="true" />
           <SunIcon v-else aria-hidden="true" />
-          {{ formatMessage(messages.changeThemeButton) }}
+          {{ formatMessage(messages.changeTheme) }}
         </button>
         <nuxt-link class="iconified-button raised-button" to="/settings">
           <SettingsIcon aria-hidden="true" />
           {{ formatMessage(commonMessages.settingsLabel) }}
         </nuxt-link>
       </div>
       <div class="not-affiliated-notice">
-        NOT AN OFFICIAL MINECRAFT SERVICE. NOT APPROVED BY OR ASSOCIATED WITH MOJANG OR MICROSOFT.
+        {{ formatMessage(footerMessages.legalDisclaimer) }}
```

Comment:
Don't think we should translate the message Microsoft requires to have since their CUG are only available in English and our crowdsourced translators are not lawyers

Reply:
per commit message:
> It wouldn't make much sense to leave it untranslated since it is meant for the end users too. It is also so small that it's unlikely to be mistranslated.

I don't see any surface for this to be mistranslated to justify dismissing this for ‘translators are not lawyers’ reason. This isn't a huge legal document. I don't see Mojang complaining about this either. If they do have a problem, they can always contact Modrinth.

Reply:
alright,we think it's ok